src/tests/utils/parse_args_test.go

```diff
@@ -0,0 +1,74 @@
+package tests
+
+import (
+	"reflect"
+	"testing"
+
+	"accretional.com/semantifly/utils"
+)
+
+func TestParseArgs(t *testing.T) {
```

Comment:
Can you add another test case for invalid input and other edge cases?

1. Empty args
2. [--flag1, --flag2]
3. [arg1, --flag1]

Reply:
done